lib/components/viewers/amenities-panel.js

```diff
@@ -0,0 +1,191 @@
+import { getCompanyIcon } from '@opentripplanner/icons/lib/companies'
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import styled from 'styled-components'
+
+import { ComponentContext } from '../../util/contexts'
+
+import RelatedPanel from './related-panel'
+
+const StyledParkAndRideIcon = styled.div`
+  background: #000;
+  border-radius: 17px;
+  color: #fff;
+  font-size: 16px;
+  font-weight: bold;
+  height: 22px;
+  line-height: 0px;
+  margin-right: 5px;
+  padding-left: 7px;
+  padding-top: 12px;
+  width: 22px;
+`
+
+const parkAndRideMarker = <StyledParkAndRideIcon>P</StyledParkAndRideIcon>
```

Comment:
The spacing on this is not quite correct. Might be worth going with an svg approach?
<img width="45" alt="Screen Shot 2021-09-14 at 11 35 44 AM" src="https://user-images.githubusercontent.com/86619099/133242796-e3e63b23-f5e1-4a4b-bc07-8279c343a80d.png">